plugins/bitbucket-cloud-common/src/BitbucketCloudClient.ts

```diff
@@ -32,9 +36,13 @@ export class BitbucketCloudClient {
     return new BitbucketCloudClient(config);
   }
 
+  private readonly fetch: BitbucketFetchFunction;
```

Comment:
This one should also accept a `fetch?: typeof fetch` in its constructor. But we'll see if typescript gets cranky with the node-fetch vs cross-fetch types.

Reply:
typescript is complaining about this one. That was the motivation for creating the extra interface.